### ISSUE TITLE
Integrity check for some files missing

### ISSUE THREAD

**Issue by onny** [owncloud/mail#1525]
Hey,
I've got the latest version of the mail app from GitHub (not the source tar ball, https://github.com/owncloud/mail/releases/download/v0.5.1/mail.tar.gz)
And it looks like there are some files with missing integrity check. I guess most of them can be removed but why are they even packaged in the first place?
```
=======
- mail
	- EXTRA_FILE
		- js/vendor/domready/.bower.json
		- js/vendor/domready/LICENSE
		- js/vendor/domready/Makefile
		- js/vendor/domready/README.md
		- js/vendor/domready/component.json
		- js/vendor/domready/make/build.js
		- js/vendor/domready/package.json
		- js/vendor/domready/ready.js
		- js/vendor/domready/ready.min.js
		- js/vendor/domready/src/ender.js
		- js/vendor/domready/tests/test.html
		- js/vendor/backbone.marionette/lib/backbone.marionette.map
		- js/vendor/backbone.marionette/lib/core/backbone.marionette.map
		- js/vendor/jquery/MIT-LICENSE.txt
		- js/vendor/jquery/src/css/swap.js
		- js/vendor/jquery/src/data/var/data_priv.js
		- js/vendor/jquery/src/sizzle/dist/sizzle.js
		- js/vendor/jquery/src/sizzle/dist/sizzle.min.js
		- js/vendor/jquery/src/sizzle/dist/sizzle.min.map
		- js/vendor/jquery/src/var/strundefined.js
```

Best regards,
Jonas

**Comment by ChristophWurst** [comment]
@Gomez any idea?

**Comment by Gomez** [comment]
Hmm, i will check it out for the next release.

**Comment by Gomez** [comment]
This was fixed with 0.5.2.
